e2e/cli-e2e/vite.config.e2e.ts

```diff
@@ -7,6 +7,7 @@ export default defineConfig({
   test: {
     reporters: ['basic'],
     testTimeout: 140_000,
+    hookTimeout: 20_000,
```

Comment:
@BioPhoton this you need to bump, I found out it was not the tests set time outing but the individual hooks because of the cleanup on windows